src/highrl/callbacks/teacher_callback.py

```diff
@@ -114,23 +114,22 @@ def _on_step(self) -> bool:
                 session = DataFrame()
 
                 for env in self.training_env.envs:  # type: ignore
-                    if len(env.session_statistics) != 0:
-                        session = concat(
-                            [session, DataFrame.from_records(env.session_statistics)]
-                        )
+                    if len(env.opt.session_statistics) != 0:
+                        stats_df = DataFrame.from_records(env.opt.session_statistics)
+                        session = concat([session, stats_df])
```

Comment:
I think a better naming for the variable stats_df would be session_stats
Also, I think it would be more readable if the variable session was plural ("sessions")

Reply:
Done